homeassistant/components/notify/sendgrid.py

```diff
@@ -31,19 +34,21 @@ def get_service(hass, config, discovery_info=None):
     api_key = config.get(CONF_API_KEY)
     sender = config.get(CONF_SENDER)
     recipient = config.get(CONF_RECIPIENT)
+    sender_name = config.get(CONF_SENDER_NAME)
```

Comment:
Use the [] syntax for required keys or ones with defaults.

Reply:
done!